etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -3,6 +3,8 @@ SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
 SPDX-License-Identifier: Apache-2.0
 */ -}}
 {{ if .Values.pachd.enterpriseLicenseKey }}
+{{- $enterpriseSecret := include "pachyderm.enterpriseSecret" . -}}
```

Comment:
I'd declare enterprise secret directly here to ensure the template isn't misused in multiple places